### PR TITLE
added canary example

### PR DESCRIPTION
**Pull request by demolab37**
first iteration

### CODE REVIEW

File: .DS_Store
Comment (cg5labs) [inline .DS_Store:1]: Please remove this file. You can add it to your .gitignore file so that it won't be picked up by git anymore.

File: main.yaml
Comment (cg5labs) [inline main.yaml:1]: Please remove the empty file from the PR.

File: .DS_Store
Comment (demolab37) [reply .DS_Store:1]: cool, thanks!

**Review by cg5labs — APPROVED** [review]

File: main.yaml
Comment (demolab37) [reply main.yaml:1]: Added .DS_Store to .gitignore

**Comment by demolab37** [comment]
thanks, let me know if any thoughts on what I had done.

**Pull request merged**